src/components/DeviceList/index.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react'
 import P from 'prop-types'
 import DeviceItem from '../DeviceItem'
+import { DevListUl } from './style.js'
 import * as S from '../DeviceItem/style.js'
 
 const DeviceList = props => {
   const { devices, onDelete, onChangeActive, onSave } = props
   return (
     <div className="deviceList">
-      <ul>
+      <DevListUl>
         <S.Wrapper isHeader>

```

Comment:
This can be named as Row instead of Wrapper and in this specific case it can be named Header that is a styled(Row) component with it's font-weight bold